src/common/styles/GlobalStyles.tsx

```diff
@@ -50,6 +50,16 @@ const GlobalStyles = createGlobalStyle`
     overscroll-behavior-x: none; /* Prevent the page redirection on swipe */
   }
 
+  ::-webkit-scrollbar {
```

Comment:
don't add it to global we just need it for tab

Reply:
Yeah, had added it for testing on the diff device first, updated now.